src/main.ts

```diff
@@ -183,10 +190,24 @@ async function moveToQuadrant(motionClient: MotionClient, armClient: ArmClient,
     try {       
       await motionClient.move(new_pose_in_frame, armName, myWorldState, constraints)
     } finally {
+      await resetJointFive(armClient)
       // homebutton().disabled = false;
     }
 }
 
+async function resetJointFive(armClient: ArmClient){
+  const currJPos = await armClient.getJointPositions()
+  console.log('currJPos: ', currJPos)
+  let jPos = currJPos.getValuesList()
+  console.log("jPos[5]: ", jPos[5])
+  if (Math.abs(jPos[5]) >= 300){
```

Comment:
I would actually do this after *every* move, not just ones where it winds up far away. I suspect this will look less jarring